src/coreclr/src/jit/unwindarm64.cpp

```diff
@@ -337,6 +622,13 @@ void Compiler::unwindSaveRegPreindexed(regNumber reg, int offset)
 
 void Compiler::unwindSaveNext()
 {
+#if defined(TARGET_UNIX)
+    if (generateCFIUnwindCodes())
+    {
+        unreached(); // do not use unwindSaveNext when generating CFI codes as there is no code for this
+    }
```

Comment:
```suggestion
    // do not use unwindSaveNext when generating CFI codes as there is no code for this
    assert(!generateCFIUnwindCodes());
```

Reply:
We are not using `unreached()` anywhere else in the JIT codebase.

Reply:
In this case you might be interested to know that this code was added based on this:
https://github.com/dotnet/runtime/blob/b4b02150b86f039f4e5637c25b7ff55e29fefb9e/src/coreclr/src/jit/unwindarm64.cpp#L228-L231

Reply:
Ah ok ... I have not realized that this is not unusual in the JIT